src/envs/blocks.py

```diff
@@ -53,19 +64,20 @@ def __init__(self) -> None:
                                   self._Holding_holds)
         self._Clear = Predicate("Clear", [self._block_type], self._Clear_holds)
         # Options
-        self._Pick = utils.SingletonParameterizedOption(
+        self._Pick: ParameterizedOption = utils.SingletonParameterizedOption(
```

Comment:
how was this not an issue before?

Reply:
it's fine without for the sake of this PR, but I'm overriding these parameterized options in pybullet env with non-singletons, so mypy complains.